helpers/string.go

```diff
@@ -25,17 +25,25 @@ func TabifyStringSlice(strs []string, tab string) []string {
 	return strs
 }
 
-// TruncateString limits the string to the given length, adding an ellipsis if the string is being truncated
+// TruncateString limits the string to the given length, adding an ellipsis if the string is being truncated,
```

Comment:
please update the unit tests in `helpers/string_test.go` to include some test cases to cover this